packages/fast-animation/tslint.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": [
-        "@microsoft/fast-tslint-rules"
+        "@microsoft/fast-tslint-rules",
+        "tslint-config-prettier"
```

Comment:
instead of putting this in each tslint.json, can we not extend the package from @microsoft/fast-tslint-rules instead?

Reply:
Synced with @nicholasrice on this. Thoughts below: 

I don't think we would *extend* this from the MSFT fast-tslint-rules because this really doesn't have anything to do with tslint. Instead of taking the dependency on this package, we would likely end up removing all style rules that conflict with prettier's opinion. This would require us to keep that manually updated instead of updating the package. Additionally, this would be breaking for anyone with a dependency on the tslint package because it would remove existing functionality they may be relying on.

If we find that this is difficult to maintain, I think we can revisit moving it.